pkg/apis/performanceprofile/v2/performanceprofile_types.go

```diff
@@ -34,6 +34,13 @@ const PerformanceProfileEnablePhysicalRpsAnnotation = "performance.openshift.io/
 // that ignores the removal of all RPS settings when realtime workload hint is explicitly set to false.
 const PerformanceProfileEnableRpsAnnotation = "performance.openshift.io/enable-rps"
 
+// PerformanceProfileAutogenerateEnablementAnnotation is an experimental annotation which let the controller
+// create the enablement file for the tech-preview `prefer-align-cpus-by-uncorecache` cpumanager policy option.
+// If the option is detected, the generated MachineConfig is automatically augmented with the expected enablement
+// file. Still, users wishing to use the feature need to explicitly opt in. To do so, instead of opting in
+// adding the enablement file, they can opt in adding the annotation and setting its value to "true".
+const PerformanceProfileAutogenerateEnablementAnnotation = "performance.openshift.io/autogenerate-enablement"
```

Comment:
I would prefer if the name was visually close to the cpu manager flag it is enabling. Something like kubeletconfig.enable-llc-feature maybe.

Reply:
fair enough, renamed